src/TLuaInterpreter.cpp

```diff
@@ -4659,6 +4659,14 @@ int TLuaInterpreter::getMudletHomeDir( lua_State * L )
     return 1;
 }
 
+int TLuaInterpreter::getMudletLuaDefaultPath( lua_State * L )
+{
+    QString path = LUA_DEFAULT_PATH "/";
```

Comment:
Look right down at the bottom of the Qt Documentation entry for `QString` this is where it is all explained! :astonished: